src/lib/i18n.ts

```diff
@@ -53,6 +53,8 @@ const i18n = createI18n({
   },
 });
 
+const userOnlyLocales = ['pt', 'ru'];
```

Comment:
I think it would be cleaner if you added a new property in the locales object for it